test/e2e/network/dual_stack.go

```diff
@@ -205,6 +206,161 @@ var _ = SIGDescribe("[Feature:IPv6DualStackAlphaFeature] [LinuxOnly]", func() {
 
 		assertNetworkConnectivity(f, *serverPods, *clientPods, "dualstack-test-client", "80")
 	})
+
+	ginkgo.It("should create service with cluster ip from primary service range [Feature:IPv6DualStackAlphaFeature:Phase2]", func() {
+		serviceName := "defaultclusterip"
+		ns := f.Namespace.Name
+		jig := e2eservice.NewTestJig(cs, serviceName)
+		defaultIPFamily := getIPFamily(framework.TestContext.IPFamily)
```

Comment:
This has changed after this https://github.com/kubernetes/kubernetes/pull/81750

you can use `framework.ClusterIsIPv6()`

Reply:
Thanks @aojea. Updated with `framework.ClusterIsIPv6()`.